test/functional/test_framework/script.py

```diff
@@ -843,17 +843,17 @@ def TaprootSignatureMsg(txTo, spent_utxos, hash_type, input_index = 0, scriptpat
 def TaprootSignatureHash(*args, **kwargs):
     return TaggedHash("TapSighash", TaprootSignatureMsg(*args, **kwargs))
 
-def taproot_tree_helper(scripts):
+def taproot_tree_helper(scripts, leaf_version):
     if len(scripts) == 0:
         return ([], bytes())
     if len(scripts) == 1:
         # One entry: treat as a leaf
         script = scripts[0]
         assert not callable(script)
         if isinstance(script, list):
-            return taproot_tree_helper(script)
+            return taproot_tree_helper(script, leaf_version)
         assert isinstance(script, tuple)
-        version = LEAF_VERSION_TAPSCRIPT
+        version = leaf_version
```

Comment:
this line should be removed and only `leaf_version` function argument should be used in the function. Both `version` and `leaf_version` will point to the same object [anyways](https://github.com/python/cpython/blob/4830f581af57dd305c02c1fd72299ecb5b090eca/Objects/longobject.c#L18-L23). Same with all  possible future leaf versions  [even 0xc0..0xfe](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#cite_note-7)

Reply:
Done in 25dc427e6c755b4c0d68599966caf95ada3565f7 , thank you for the review.